bitnami/nginx/values.yaml

```diff
@@ -44,6 +44,19 @@ service:
   ##
   externalTrafficPolicy: Cluster
 
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  path: /
+  hosts:
+    - chart-example.local
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
```

Comment:
Try to follow the standard we used on other charts: https://github.com/bitnami/charts/blob/master/upstreamed/wordpress/values.yaml#L206

Reply:
done. Followed standard set in wordpress chart